db_migration/db_migration_non-single.sh

```diff
@@ -0,0 +1,37 @@
+# This code calls db migration.
+
+# To use the db migration, a klaytn client should not be running.
+# You need to have access to the DB.
+# To checkout migration status, `tail -f logs-body.out`
+
+# BIN file
+KLAYTN_BIN=/Users/mini-admin/go/src/github.com/klaytn/klaytn/build/bin/ken
```

Comment:
```suggestion
KLAYTN_BIN=~/klaytn/bin/ken
```

How about updating this path in accordance with the `db_migration.sh`?